NEWS.md

```diff
@@ -1,3 +1,7 @@
+## tibble 1.3.0.9001 (2017-04-18)
+
+- `tribble` now handles columns with class andles other columns with a class. (#161, @NikNakk)
```

Comment:
Fair enough. Have reverted to previous NEWS.md. I was following what Hadley had asked me to do for previous PR to `dplyr`.